database/mongo.js

```diff
@@ -1,10 +1,20 @@
 var mongoose = require('mongoose');
+const AutoIncrement = require('mongoose-sequence')(mongoose);
 mongoose.connect('mongodb://localhost/topBunk');
 
 var db = mongoose.connection;
-db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', function() {
-  
-  module.exports = db;
 
-});
+var listingSchema = new mongoose.Schema({
+  _id: { type: Number, unique: true },
+  price: Number,
+  maxGuests: Number,
+  minStay: Number,
+  stars: mongoose.Decimal128,
+  numRatings: Number,
+  bookedDates: [String]
+}, { _id: false });
+listingSchema.plugin(AutoIncrement);
```

Comment:
@wyatt-troia, do you need to add an option here that will tell mongoose-sequence which counter to use so that it will begin auto-incrementing at 10,000,000? Or is that handled in your database initializing schema?

Reply:
I have an NPM script that will pipe my `updateCounter.js `file to Mongo, which will update mongoose-sequence to begin counting at 10,000,001. I could not find that kind of option exists for mongoose-sequence.

Reply:
Ok, everything else looks good. I'll approve this pull request.